pallets/gear/src/lib.rs

```diff
@@ -303,60 +302,67 @@ pub mod pallet {
             Ok(message)
         }
 
-        pub fn get_gas_spent(dest: H256, payload: Vec<u8>) -> Option<u64> {
-            let mut ext_manager = ExtManager::<T>::default();
+        pub fn get_gas_spent(
+            source: H256,
+            dest: H256,
```

Comment:
who do you even use this, given you have just program code?

Reply:
Assumed to get the code from the storage, but replaced it by sending the code as a parameter.